plugin/trino-raptor-legacy/src/test/java/io/trino/plugin/raptor/legacy/BaseRaptorConnectorTest.java

```diff
@@ -111,16 +112,16 @@ public void testRenameTableAcrossSchema()
 
         String schemaName = "test_schema_" + randomTableSuffix();
 
-        String renamedTable = schemaName + ".test_rename_new_" + randomTableSuffix();
-        assertUpdate("ALTER TABLE " + tableName + " RENAME TO " + renamedTable);
+        String renamedTable = "test_rename_new_" + randomTableSuffix();
+        assertUpdate(format("ALTER TABLE %s RENAME TO %s.%s", tableName, schemaName, renamedTable));
```

Comment:
Unrelated (should be separate commit)

Also, while `format` is better for general string manipulation, direct `+` concatenation allows IntelliJ to apply syntax highlighting. At least my IntelliJ version does SQL syntax highlihting for the code before the change, and doesn't do after.
So maybe just drop the change?

Reply:
I don't have a strong opinion, but I feel 3 concatenation for simple SQL is less readable than `format` even if the former has syntax highlight. Anyway, reverted to the original style. 
```java
"ALTER TABLE " + tableName + " RENAME TO " + schemaName + "." + renamedTable

format("ALTER TABLE %s RENAME TO %s.%s", tableName, schemaName, renamedTable)
```